typings/lhr-lite.d.ts

```diff
@@ -0,0 +1,132 @@
+        score: number;
+        /** An array of references to all the audit members of this category. */
+        auditRefs: AuditRef[];
+        /** An optional description for manual audits within this category. */
+        manualDescription?: string;
+      }
+
+      /**
+       * A reference to an audit result, with weighting and grouping information
+       * for its place in this category.
+       */
+      export interface AuditRef {
+        /** Matches a key in the top-level `audits` object. */
+        auditId: string;
+        /** The weight of the audit's score in the overall category score. */
+        weight: number;
+        /** Optional grouping within the category. Matches the key in the top-level `categoryGroups` object. */
+        groupId?: string;
+      }
+
+      export interface CategoryGroup {
+        /** The human-friendly name of the category. */
+        title: string;
+        /** A brief description of the purpose of the display group. */
+        description: string;
+      }
+
+      export interface Audit {
+        /** The brief description of the audit. The text can change depending on if the audit passed or failed. */
+        title: string;
+        /** A more detailed description that describes why the audit is important and links to Lighthouse documentation on the audit; markdown links supported. */
+        description: string;
+        /** The scored value determined by the audit, in the range `0-1`. */
+        score: number;
+        /**
+         * A string identifying how the score should be interpreted:
+         * 'binary': pass/fail audit (0 and 1 are only possible scores).
+         * 'numeric': scores of 0-1 (map to displayed scores of 0-100).
+         * 'informative': the audit is an FYI only, and can't be interpreted as pass/fail. Ignore the score.
+         * 'notApplicable': the audit turned out to not apply to the page. Ignore the score.
+         * 'manual': The audit exists only to tell you to review something yourself. Ignore the score.
+         */
+        scoreDisplayMode: 'binary' | 'numeric' | 'informative' | 'notApplicable' | 'manual';
+        /** Extra information provided by some types of audits. */
+        details?: AuditDetails;
+        /** Error message from any exception thrown while running this audit. */
+        errorMessage?: string;
+      }
+
+      export type AuditDetails = AuditMetricDetails | AuditOpportunityDetails | AuditTableDetails;
+
+      export interface AuditMetricDetails {
+        type: 'metric';
+        /** The value of the metric expressed in milliseconds. */
+        timespanMs?: number;
+      }
+
+      export interface AuditOpportunityDetails {
+        type: 'opportunity';
+        wastedMs: number
+        wastedBytes?: number
+        headings: AuditColumnHeading[];
+        rows: AuditTableRow[];
+      }
+
+      export interface AuditTableDetails {
+        type: 'table';
```

Comment:
do we actually have any of these that we want to expose? seems like we're only wanting to expose the opportunities which all use the table headings/rows types